### ISSUE TITLE
Getting compiler error on archiving for Apollo 0.10.1 version

### ISSUE THREAD

**Issue by pranita-mindstix** [apollographql/apollo-ios#594]
I have installed the latest Apollo - '0.10.1' for my iOS project. Building and running of an application work perfectly. But getting a compile-time error while archiving the project :(
Attached screenshot for error.

<img width="1110" alt="Screenshot 2019-06-28 at 3 38 49 PM" src="https://user-images.githubusercontent.com/8039894/60336259-81f9a100-99bd-11e9-9572-a0c6e9357fd8.png">

I am using - 
Xcode - 10.2.1
Swift - 5.0.1
And script I am using for generating Apollo graphql API file is - 

_APOLLO_FRAMEWORK_PATH="$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 1)"

if [ -z "$APOLLO_FRAMEWORK_PATH" ]; then
echo "error: Couldn't find Apollo.framework in FRAMEWORK_SEARCH_PATHS; make sure to add the framework to your project."
exit 1
fi

cd "${SRCROOT}/${TARGET_NAME}"
$APOLLO_FRAMEWORK_PATH/check-and-run-apollo-cli.sh codegen:generate --queries="$(find . -name '*.graphql')" --schema=schema.json API.swift_

**Comment by designatednerd** [comment]
@pranita-mindstix Weird that it works for build + run but not for archive - have you tried altering the `-maxdepth` parameter to something else (or omitting it altogether)?

**Comment by pranita-mindstix** [comment]
> @pranita-mindstix Weird that it works for build + run but not for archive - have you tried altering the `-maxdepth` parameter to something else (or omitting it altogether)?

**Comment by pranita-mindstix** [comment]
@designatednerd Sorry, I might be asking a stupid question, but what to change for maxdepth parameter? I mean do I really need to change this script?

**Comment by designatednerd** [comment]
No such thing as a stupid question! 

So where it currently says: 

```
_APOLLO_FRAMEWORK_PATH="$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 1)"
```

I would try updating it to something like

```
_APOLLO_FRAMEWORK_PATH="$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 10)"
```

My suspicion is that `maxdepth` is not looking far enough down the tree to find the framework. If that doesn't work then something else is going on.

**Comment by pranita-mindstix** [comment]
Nope, altering maxdepth didn't work.. :(

**Comment by designatednerd** [comment]
:sad-trombone: - can you let me know what you're using to integrate - cocoapods, carthage, or manual? I'll see if i can repro.

**Comment by pranita-mindstix** [comment]
I am using cocoapods and I am on the version - 1.5.3

**Comment by designatednerd** [comment]
@pranita-mindstix Can you try replacing your script with the following:

```
SCRIPT_PATH="${PODS_ROOT}/Apollo/scripts"
cd "${SRCROOT}/${TARGET_NAME}"
"${SCRIPT_PATH}"/check-and-run-apollo-cli.sh codegen:generate --queries="$(find . -name '*.graphql')" --schema=schema.json API.swift
```

**Comment by pranita-mindstix** [comment]
@designatednerd I have replaced the script with your suggested code. Able to archive now.
Thanks for your quick help. :) 👍 
One more question, issue was with the path which we were passing for getting check-and-run-apollo-cli.sh script file, correct?

**Comment by designatednerd** [comment]
Basically, yes - there have been some recent changes which made it better to go with the method you're using here, but I have a sneaking suspicion there was a space in a directory somewhere that was making Xcode freak out, which this should (theoretically) alleviate

**Comment by designatednerd** [comment]
Closing this out - updated instructions will be released with 0.11.0
